### ISSUE TITLE
ara-manage runserver silently hangs when Postgres dependency is missing

### ISSUE THREAD

**Issue by zxaos** [ansible-community/ara#63]
## What component is this about ?
API

## What is your ARA installation like ?
Installed from PyPi

## What is happening ?
If `ara-manage runserver` is invoked while the db engine set to Postgres but without the `psycopg2` or `psycopg2-binary` package installed, it fails to start, but doesn't print any information about why it's not working.

Output:
```
2019-08-07 20:30:30,855 INFO django.utils.autoreload: Watching for file changes with StatReloader
```

## What should be happening ?
Something closer to what happens if `ara-manage` is called without arguments:
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/django/db/backends/postgresql/base.py", line 20, in <module>
    import psycopg2 as Database
ModuleNotFoundError: No module named 'psycopg2'

During handling of the above exception, another exception occurred:
...
```

**Comment by dmsimard** [comment]
@zxaos thanks for the report ! It looks like the right place to handle this exception would be here: https://github.com/ansible-community/ara/blob/master/ara/server/__main__.py

**Comment by dmsimard** [comment]
This will be fixed in 1.3 by https://review.opendev.org/#/c/695019/
